app/guid-node/metadata/-components/metadata-tab-view/component.ts

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import Media from 'ember-responsive';
+import Analytics from 'ember-osf-web/services/analytics';
+import CedarMetadataRecordModel from 'ember-osf-web/models/cedar-metadata-record';
+
+interface TabViewArgs {
+    cedarMetadataRecord: CedarMetadataRecordModel;
+    id: number;
+    activeId: number;
+}
+
+
+export default class MetadataTabView extends Component<TabViewArgs> {
+    @service media!: Media;
+    @service analytics!: Analytics;
+
+    cedarMetadataRecord = this.args.cedarMetadataRecord;
+
+    private logAnalytics(): void {
+        // eslint-disable-next-line max-len
+        this.analytics.click('tab', `Metadata tab - Change tab to: ${this.cedarMetadataRecord.template.get('schemaName')}`);
+    }
+
+    get isActive(): boolean {
+        if (this.args.id+ 1 === this.args.activeId) {
+            this.logAnalytics();
```

Comment:
I believe this implementation for analytics is going to lead to faulty data, as a route being active doesn't necessarily mean that a user clicked a tab (e.g. the `OSF` tab is going to be the tab selected by default, but users may not have clicked that tab if they were automatically routed). I think a more robust/correct way to implement this would be to add a `data-analytics-name="Change tab to: ...` to the `<tablist.tab>` elements in `metadata-tabs` and remove the call to `this.logAnalytics()` here